### ISSUE TITLE
Evaluation?

### ISSUE THREAD

**Issue by RAMIREDDYDEVARAM** [ZheC/Realtime_Multi-Person_Pose_Estimation#68]
Hello.
I am new to this field. I need some guidance to evaluation. Recently I finished training with the same data until max 600000 iterations. May I know how to evaluate the results as like you published in the paper. I am just practicing. It's may useful for my further research. I mean what we need to evaluate our model and how to arrange the Matlab Code.
I am really appreciate if guide me. Thanking you so much.

**Comment by ZheC** [comment]
What I did is to create a small validation set (so it is not in the training set) before training. I will evaluate the trained models in the evaluation set and pick up the one with highest accuracy. The accuracy will be either COCO eval code (https://github.com/pdollar/coco) or MPII eval code (http://human-pose.mpi-inf.mpg.de/#evaluation).
